docs/intentReaderHandlers/HealthCheckHandler.md

```diff
@@ -0,0 +1,110 @@
+# HealthCheckHandler 
+
+## Overview
+
+HealthCheckHandler is invoked when the task specified in the intent has the value "healthCheck". This handler is responsible for sending the healthCheck command and validating whether FCA is launched as a third-party app or not.
+
+It performs the following actions:
+1. Parses the input message received.
+2. Confirms the healthCheck status.
+3. Saves the response or error.
+4. Formats the result and sends the response back to IntentReader.
+
+## Usage
+
+* Request Format
+
+```json
+    {
+        "action": "search",
+        "data": {
+            "query": {
+                "task": "healthCheck",
+                "action": "NA",
+                "appType": "<appType>"
+            }
+        },
+        "context": {
+            "source": "device"
+        }
+    }
+```
+
+### Parameters
+
+| Key               | Description                                                                       | Required? |
+|-------------------|-----------------------------------------------------------------------------------|-----------|
+| task              | "healthCheck"- Its a static value and should not be changed for this handler      | Y         |
+| appType           | Corresponding intent is launching on which app                                    | Y         |
+
+
+* Response Format
+* Response can be either "true" or an error response
+
+```json
+    true
+```
+### Parameters
+
+| Key                         | Description                                                          |
+| --------------------------- | ---------------------------------------------------------------------|
+| true/null                   | It indicates whether FCA is launched as third party app or not       |
```

Comment:
You say the Response can be either "true" or an error response but here you say true/null - does null vs. error indicate a different behavior.